Cargo.toml

```diff
@@ -28,7 +27,7 @@ curl = { version = "0.4.21", features = ['http2'] }
 curl-sys = "0.4.18"
 env_logger = "0.6.0"
 pretty_env_logger = { version = "0.3", optional = true }
-failure = "0.1.5"
+failure = { version = "0.1.5", default_features = false }
```

Comment:
I don't think this will work for multiple reasons:

1. cargo uses the derive feature in several places.
2. rustfix uses failure, so this has no effect.
3. The `std` feature is also used.
4. Even if it were possible, I think rust-lang/rust enforces tool dependencies to use the same features, so this would have to ripple out to all other users (like rls, etc.).